chocolatey-community/chocolatey-community-validation
# (#58) Implement requirement rule CPMR0074

## Description Of Changes

This introduces the latest requirement rule that was added in Package
Validator related to verifying that hook packages has not been used as a dependency.

## Motivation and Context

We want to have the rules implemented in Package Validator implemented in the community extension.

## Testing

1. Create a new package nuspec using `choco new`.
2. Update the `dependencies` list to include a dependency on a package ending with `.hook`.
3. Run `choco pack` against the nuspec.
4. Verify the rule `CPMR0074` is flagged.

### Operating Systems Testing

- Windows 10

## Change Types Made



## Change Checklist


## Related Issue


Fixes #58
